CHANGELOG.md

```diff
@@ -1,92 +1,116 @@
-# CHANGELOG
-
-v2.10.0 (01.04.2024)
---------------------
-- Add support **JSON** columns in **orderBy** statement by @msmakouz (#184)
-- Add `mediumText` column type by @msmakouz (#178)
-- Fix caching of SQL insert query with Fragment values by @msmakouz (#177)
-- Fix detection of enum values in PostgreSQL when a enum field has only one value by @msmakouz (#181)
-
-v2.9.0 (27.03.2024)
--------------------
-- Added `logInterpolatedQueries` for detailed query logging and refined `logQueryParameters` to accurately log query parameters as arrays, enhancing ORM debugging and monitoring. @lotyp (#165)
-- Improved logging with enriched context in Driver.php, including driver details and query parameters. @lotyp (#165)
-- Improved `orderBy` method to accept a null direction parameter, providing more flexibility when constructing queries by @butschster (#167)
-- Added support for PostgreSQL-specific directions **NULLS FIRST** and **NULLS LAST** for more control over null value ordering in result sets by @butschster (#167)
-
-v2.8.1 (08.02.2024)
--------------------
-- Fix compiling of Fragment with parameters in the returning() definition by @msmakouz (#161)
-- Fix comparison of Fragment in Column default value by @msmakouz (#162)
-
-v2.8.0 (08.02.2024)
--------------------
-- The `withDatetimeMicroseconds` option now affects the interpolator by @msmakouz (#155)
-- Postgres: support for multiple returning columns by @roxblnfk and @msmakouz (#157)
-- MSSQL: support for multiple returning columns by @msmakouz (#160)
-
-v2.7.1 (22.12.2023)
--------------------
-- Fix setting `precision` and `scale` through attributes by @msmakouz (#148)
-- Fix quoting with an underscore at the beginning by @msmakouz (#151)
-- Fix behavior of the `Column::type()` method by adding default size to column type mappings by @msmakouz (#150)
-
-v2.7.0 (04.12.2023)
--------------------
-- Add `varbinary` support in MySQL; optimize `size` attribute by @msmakouz (#146)
-- Add the ability to use WHERE IN and WHERE NOT IN with array values
-  The value sequence may contain `FragmentInterface` objets by @msmakouz and @roxblnfk (#147)
-
-v2.6.0 (02.11.2023)
--------------------
-- Fix incorrect parameters processing for JOIN subqueries by @smelesh (#133)
+# Changelog
+
+## [2.10.0](https://github.com/cycle/database/compare/2.9.0...v2.10.0) (2024-04-01)
+
+### Features
+* Added support JSON columns in orderBy statement by @msmakouz (#184)
+* Added mediumText column type by @msmakouz (#178)
```

Comment:
```suggestion
- Added support JSON columns in orderBy statement by @msmakouz (#184)
- Added mediumText column type by @msmakouz (#178)
```

Why there are `*` instead of `-`?

Reply:
release-please auto generates and probably uses that kind of syntax by default